src/App.tsx

```diff
@@ -15,71 +16,91 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+type State = {
+  selectedGood: string;
+};
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+export class App extends React.Component<{}, State> {
+  state: Readonly<State> = {
+    selectedGood: 'Jam',
+  };
 
-      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+  clearButtonHandler = () => {
+    this.setState({ selectedGood: '' });
+  };
```

Comment:
much clearer this way
```suggestion
  clearSelection = () => {
    this.setState({ selectedGood: '' });
  };
```

Reply:
fixed 👌